tests/behaviour/steps/connection/mod.rs

```diff
@@ -7,30 +7,35 @@
 use std::{
     error::Error,
     fmt,
-    sync::{Arc, Mutex, OnceLock},
+    sync::{Arc, Mutex},
 };
 
 use macro_rules_attribute::apply;
 use server::{parameters::config::Config, typedb};
 use test_utils::{create_tmp_dir, TempDir};
+use tokio::{runtime::Runtime, sync::OnceCell};
 
 use crate::{generic_step, Context};
 
 mod database;
 mod transaction;
 
-static TYPEDB: OnceLock<(TempDir, Arc<Mutex<typedb::Server>>)> = OnceLock::new();
+const DISTRIBUTION: &str = "TypeDB TEST";
+static TYPEDB: OnceCell<(TempDir, Arc<Mutex<typedb::Server>>)> = OnceCell::const_new();
```

Comment:
`Server::open` is async now, so I need to pass an `async` closure into `get_or_init`.